packages/react/src/auto/AutoForm.ts

```diff
@@ -58,7 +58,14 @@ export const useFormFields = (
   return useMemo(() => {
     if (!metadata) return [];
     const action = isActionMetadata(metadata) ? metadata.action : metadata;
-    const objectFields = action.inputFields.filter((field) => field.configuration.__typename === "GadgetObjectFieldConfig");
+    const objectFields =
+      metadata.__typename === "GadgetGlobalAction"
```

Comment:
It seems that the intention with this to check if the metadata for this is modelAction/GlobalAction. That overlaps a bit with the above `isActionMetadata(metadata)` call so you can add this check there and then store that result as a const